From 86c1494e9c91ef0ea89bcce3334ffa2395ed696a Mon Sep 17 00:00:00 2001
From: Austin Morton <austin.morton@aquatic.com>
Date: Tue, 2 Sep 2025 06:30:02 -0500
Subject: [PATCH 1/2] Add ray_systemlibs

---
 WORKSPACE | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/WORKSPACE b/WORKSPACE
index e56c6a109b..b4cbb29097 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -28,6 +28,10 @@ load("//bazel:ray_deps_setup.bzl", "ray_deps_setup")
 
 ray_deps_setup()
 
+load("//:thirdparty/systemlibs/repo.bzl", "ray_systemlibs")
+
+ray_systemlibs()
+
 load("//bazel:ray_deps_build_all.bzl", "ray_deps_build_all")
 
 ray_deps_build_all()
-- 
2.43.0

