From 12f98ac3a15fe2600c73acf0eac4a4d07b53a162 Mon Sep 17 00:00:00 2001
From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
Date: Tue, 10 Nov 2020 23:26:35 +0300
Subject: [PATCH 01/10] Disable making entry scripts

---
 python/setup.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index a04fa86b46..38b5984b69 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -833,9 +833,9 @@ if __name__ == "__main__":
         extras_require=setup_spec.extras,
         entry_points={
             "console_scripts": [
-                "ray=ray.scripts.scripts:main",
-                "tune=ray.tune.cli.scripts:cli",
-                "serve=ray.serve.scripts:cli",
+                #"ray=ray.scripts.scripts:main",
+                #"tune=ray.tune.cli.scripts:cli",
+                #"serve=ray.serve.scripts:cli",
             ]
         },
         package_data={
-- 
2.52.0

