From 762223c2a4df0c861c4a1248ac32328e5cbaf11a Mon Sep 17 00:00:00 2001
From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
Date: Tue, 10 Nov 2020 23:26:35 +0300
Subject: [PATCH 2/5] Disable making non-core entry scripts

---
 python/setup.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index 15d6d7a8a..1b8c8e156 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -777,10 +777,10 @@ setuptools.setup(
     entry_points={
         "console_scripts": [
             "ray=ray.scripts.scripts:main",
-            "rllib=ray.rllib.scripts:cli [rllib]",
-            "tune=ray.tune.cli.scripts:cli",
-            "ray-operator=ray.ray_operator.operator:main",
-            "serve=ray.serve.scripts:cli",
+            #"rllib=ray.rllib.scripts:cli [rllib]",
+            #"tune=ray.tune.cli.scripts:cli",
+            #"ray-operator=ray.ray_operator.operator:main",
+            #"serve=ray.serve.scripts:cli",
         ]
     },
     package_data={
-- 
2.38.1.windows.1

