From 5452856effbc8356f2feb5697624a98e8b8251e1 Mon Sep 17 00:00:00 2001
From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
Date: Thu, 5 Nov 2020 17:05:44 +0300
Subject: [PATCH 2/7] Fix redis build for non-default compiler drivers

Signed-off-by: Vasily Litvinov <vasilij.n.litvinov@intel.com>
---
 bazel/BUILD.redis | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/bazel/BUILD.redis b/bazel/BUILD.redis
index 9cd2e7af9..9ec69d433 100644
--- a/bazel/BUILD.redis
+++ b/bazel/BUILD.redis
@@ -14,7 +14,7 @@ genrule(
         "redis-cli",
     ],
     cmd = """
-        unset CC LDFLAGS CXX CXXFLAGS
+        export CC=$(CC)
         tmpdir="redis.tmp"
         p=$(location Makefile)
         cp -p -L -R -- "$${p%/*}" "$${tmpdir}"
@@ -28,6 +28,7 @@ genrule(
         rm -r -f -- "$${tmpdir}"
     """,
     visibility = ["//visibility:public"],
+    toolchains = ["@bazel_tools//tools/cpp:current_cc_toolchain"],
 )
 
 # This library is for internal hiredis use, because hiredis assumes a
-- 
2.32.0.windows.2

