From 698bc58efaa0603b3209d55bd3890644428fb369 Mon Sep 17 00:00:00 2001
From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
Date: Tue, 10 Nov 2020 23:26:35 +0300
Subject: [PATCH 2/4] Disable making entry scripts

---
 python/setup.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index b48c1510d0..26fa74e229 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -787,10 +787,10 @@ setuptools.setup(
     extras_require=setup_spec.extras,
     entry_points={
         "console_scripts": [
-            "ray=ray.scripts.scripts:main",
-            "rllib=ray.rllib.scripts:cli [rllib]",
-            "tune=ray.tune.cli.scripts:cli",
-            "serve=ray.serve.scripts:cli",
+            #"ray=ray.scripts.scripts:main",
+            #"rllib=ray.rllib.scripts:cli [rllib]",
+            #"tune=ray.tune.cli.scripts:cli",
+            #"serve=ray.serve.scripts:cli",
         ]
     },
     package_data={
