From 4d2046a891d75eb52f0837fe41065756dc2cf642 Mon Sep 17 00:00:00 2001
From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
Date: Fri, 9 May 2025 13:12:32 +0300
Subject: [PATCH 2/9] Ignore warnings in event.cc and logging.cc

---
 .bazelrc | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/.bazelrc b/.bazelrc
index 13774dda7c..575538c5f1 100644
--- a/.bazelrc
+++ b/.bazelrc
@@ -60,6 +60,9 @@ build:clang-cl --per_file_copt="-\\.(asm|S)$@-Werror"
 # Ignore warnings for protobuf generated files and external projects.
 build --per_file_copt="\\.pb\\.cc$@-w"
 build --per_file_copt="src/ray/thirdparty/.*$@-w"
+# Ignore one specific warning
+build --per_file_copt="event\\.cc$@-w"
+build --per_file_copt="logging\\.cc$@-w"
 build:linux --per_file_copt="-\\.(asm|S)$,external/.*@-w,-Wno-error=implicit-function-declaration,-Wno-error=unused-function"
 build:macos --per_file_copt="-\\.(asm|S)$,external/.*@-w,-Wno-error=implicit-function-declaration,-Wno-error=unused-function,-Wno-missing-template-arg-list-after-template-kw"
 # Ignore warnings for host tools, which we generally can't control.
-- 
2.43.0

