From 9c56c8b45e86273f7ffe9e2e5bbf9b8321599048 Mon Sep 17 00:00:00 2001
From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
Date: Sun, 19 Jan 2025 22:56:08 +1100
Subject: [PATCH 2/8] Ignore warnings in event.cc and logging.cc

---
 .bazelrc | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/.bazelrc b/.bazelrc
index 13774dda7c..7db790a4bf 100644
--- a/.bazelrc
+++ b/.bazelrc
@@ -57,6 +57,9 @@ build:clang-cl --per_file_copt="-\\.(asm|S)$@-Werror"
 build:msvc-cl     --per_file_copt="-\\.(asm|S)$@-WX"
 # Ignore warnings for protobuf generated files and external projects.
 build --per_file_copt="\\.pb\\.cc$@-w"
+# Ignore one specific warning
+build --per_file_copt="event\\.cc$@-w"
+build --per_file_copt="logging\\.cc$@-w"
 build:linux --per_file_copt="-\\.(asm|S)$,external/.*@-w,-Wno-error=implicit-function-declaration,-Wno-error=unused-function"
 build:macos --per_file_copt="-\\.(asm|S)$,external/.*@-w,-Wno-error=implicit-function-declaration,-Wno-error=unused-function,-Wno-missing-template-arg-list-after-template-kw"
 # Ignore warnings for host tools, which we generally can't control.
