From db42a42d6116cdde036aac4a301664357f0998d7 Mon Sep 17 00:00:00 2001
From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
Date: Tue, 10 Nov 2020 23:26:35 +0300
Subject: [PATCH 2/5] Disable making entry scripts

---
 python/setup.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index d1566cf14f..c79aca8689 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -795,10 +795,10 @@ setuptools.setup(
     extras_require=setup_spec.extras,
     entry_points={
         "console_scripts": [
-            "ray=ray.scripts.scripts:main",
-            "rllib=ray.rllib.scripts:cli [rllib]",
-            "tune=ray.tune.cli.scripts:cli",
-            "serve=ray.serve.scripts:cli",
+            #"ray=ray.scripts.scripts:main",
+            #"rllib=ray.rllib.scripts:cli [rllib]",
+            #"tune=ray.tune.cli.scripts:cli",
+            #"serve=ray.serve.scripts:cli",
         ]
     },
     package_data={
-- 
2.39.2 (Apple Git-143)

