From 092468d20a20d4ee8da4d256ab3aa2b53645c4ca Mon Sep 17 00:00:00 2001
From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
Date: Thu, 5 Nov 2020 17:09:46 +0300
Subject: [PATCH 03/14] Fix plasma linking for ancient glibc

Signed-off-by: Vasily Litvinov <vasilij.n.litvinov@intel.com>
---
 BUILD.bazel | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/BUILD.bazel b/BUILD.bazel
index 879633444..0463f7c9d 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -223,6 +223,7 @@ PLASMA_LINKOPTS = [] + select({
         "-DefaultLib:" + "ws2_32.lib",
     ],
     "//conditions:default": [
+        "-lrt",
     ],
 })
 
@@ -315,6 +316,7 @@ cc_binary(
         "src/ray/plasma/store_exec.cc",
     ],
     copts = PLASMA_COPTS,
+    linkopts = PLASMA_LINKOPTS,
     visibility = ["//visibility:public"],
     deps = [
         ":plasma_store_server_lib",
-- 
2.28.0

