From e0ec9c68e7b5f060e29cfd5e33579bb9700d63af Mon Sep 17 00:00:00 2001
From: Kai Fricke <kai@anyscale.com>
Date: Fri, 3 Dec 2021 10:55:23 -0800
Subject: [PATCH 3/5] Remove all dependencies from setup.py

Signed-off-by: Jiajun Yao <jeromeyjj@gmail.com>
Signed-off-by: Matti Picus <matti.picus@gmail.com>
---
 python/setup.py | 80 ++++++-------------------------------------------
 1 file changed, 9 insertions(+), 71 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index 1b8c8e156..c6ee62a96 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -211,43 +211,13 @@ ray_files += [
 # also update the matching section of requirements/requirements.txt
 # in this directory
 if setup_spec.type == SetupType.RAY:
-    if sys.version_info >= (3, 7):
-        pandas_dep = "pandas >= 1.3"
-        numpy_dep = "numpy >= 1.20"
-    else:
-        # Pandas dropped python 3.6 support in 1.2.
-        pandas_dep = "pandas >= 1.0.5"
-        # Numpy dropped python 3.6 support in 1.20.
-        numpy_dep = "numpy >= 1.19"
     setup_spec.extras = {
-        "data": [
-            numpy_dep,
-            pandas_dep,
-            "pyarrow >= 6.0.1, < 7.0.0",
-            "fsspec",
-        ],
-        "default": [
-            # If adding dependencies necessary to launch the dashboard api server,
-            # please add it to dashboard/optional_deps.py as well.
-            "aiohttp >= 3.7",
-            "aiohttp_cors",
-            "colorful",
-            "py-spy >= 0.2.0",
-            "requests",
-            "gpustat >= 1.0.0",  # for windows
-            "opencensus",
-            "pydantic",
-            "prometheus_client >= 0.7.1, < 0.14.0",
-            "smart_open",
-        ],
-        "serve": ["uvicorn", "requests", "starlette", "fastapi", "aiorwlock"],
-        "tune": ["pandas", "tabulate", "tensorboardX>=1.9", "requests"],
-        "k8s": ["kubernetes", "urllib3"],
-        "observability": [
-            "opentelemetry-api==1.1.0",
-            "opentelemetry-sdk==1.1.0",
-            "opentelemetry-exporter-otlp==1.1.0",
-        ],
+        "data": [],
+        "default": [],
+        "serve": [],
+        "tune": [],
+        "k8s": [],
+        "observability": [],
     }
 
     # Ray Serve depends on the Ray dashboard components.
@@ -261,18 +231,7 @@ if setup_spec.type == SetupType.RAY:
     if sys.version_info >= (3, 7, 0):
         setup_spec.extras["k8s"].append("kopf")
 
-    setup_spec.extras["rllib"] = setup_spec.extras["tune"] + [
-        "dm_tree",
-        "gym>=0.21.0,<0.24.0",
-        "lz4",
-        # matplotlib (dependency of scikit-image) 3.4.3 breaks docker build
-        # Todo: Remove this when safe?
-        "matplotlib!=3.4.3",
-        "scikit-image",
-        "pyyaml",
-        "scipy",
-    ]
-
+    setup_spec.extras["rllib"] = setup_spec.extras["tune"]
     setup_spec.extras["train"] = setup_spec.extras["tune"]
 
     # Ray AI Runtime should encompass Data, Tune, and Serve.
@@ -297,28 +256,7 @@ if setup_spec.type == SetupType.RAY:
 # install-core-prerelease-dependencies.sh so we can test
 # new releases candidates.
 if setup_spec.type == SetupType.RAY:
-    setup_spec.install_requires = [
-        "attrs",
-        "click >= 7.0, <= 8.0.4",
-        "dataclasses; python_version < '3.7'",
-        "filelock",
-        "grpcio >= 1.32.0; python_version < '3.10'",
-        "grpcio >= 1.42.0; python_version >= '3.10'",
-        "jsonschema",
-        "msgpack >= 1.0.0, < 2.0.0",
-        "numpy >= 1.16; python_version < '3.9'",
-        "numpy >= 1.19.3; python_version >= '3.9'",
-        "packaging; python_version >= '3.10'",
-        "protobuf >= 3.15.3, != 3.19.5",
-        "pyyaml",
-        "aiosignal",
-        "frozenlist",
-        "requests",
-        # Light weight requirement, can be replaced with "typing" once
-        # we deprecate Python 3.7 (this will take a while).
-        "typing_extensions; python_version < '3.8'",
-        "virtualenv>=20.0.24",  # For pip runtime env.
-    ]
+    setup_spec.install_requires = []
 
 
 def is_native_windows_or_msys():
@@ -772,7 +710,7 @@ setuptools.setup(
     # The BinaryDistribution argument triggers build_ext.
     distclass=BinaryDistribution,
     install_requires=setup_spec.install_requires,
-    setup_requires=["cython >= 0.29.26", "wheel"],
+    setup_requires=[],
     extras_require=setup_spec.extras,
     entry_points={
         "console_scripts": [
-- 
2.38.1.windows.1

