From a5a4411a7ad8ca1dcd3ffcb1175340923e289da6 Mon Sep 17 00:00:00 2001
From: mattip <matti.picus@gmail.com>
Date: Mon, 25 Dec 2023 12:13:06 +0200
Subject: [PATCH 4/7] Remove all dependencies from setup.py

---
 python/setup.py | 121 +++++-------------------------------------------
 1 file changed, 12 insertions(+), 109 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index bbf74a8b45..0aa524af1b 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -225,103 +225,18 @@ ray_files += [
 # also update the matching section of requirements/requirements.txt
 # in this directory
 if setup_spec.type == SetupType.RAY:
-    pandas_dep = "pandas >= 1.3"
-    numpy_dep = "numpy >= 1.20"
-    if sys.platform != "win32":
-        pyarrow_dep = "pyarrow >= 6.0.1"
-    else:
-        # Serialization workaround for pyarrow 7.0.0+ doesn't work for Windows.
-        pyarrow_dep = "pyarrow >= 6.0.1, < 7.0.0"
     setup_spec.extras = {
-        "data": [
-            numpy_dep,
-            pandas_dep,
-            pyarrow_dep,
-            "fsspec",
-        ],
-        "default": [
-            # If adding dependencies necessary to launch the dashboard api server,
-            # please add it to dashboard/optional_deps.py as well.
-            "aiohttp >= 3.7",
-            "aiohttp_cors",
-            "colorful",
-            "py-spy >= 0.2.0",
-            "requests",
-            "gpustat >= 1.0.0",  # for windows
-            "grpcio >= 1.32.0; python_version < '3.10'",  # noqa:E501
-            "grpcio >= 1.42.0; python_version >= '3.10'",  # noqa:E501
-            "opencensus",
-            "pydantic!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,<3",
-            "prometheus_client >= 0.7.1",
-            "smart_open",
-            "virtualenv >=20.0.24, !=20.21.1",  # For pip runtime env.
-        ],
-        "client": [
-            # The Ray client needs a specific range of gRPC to work:
-            # Tracking issues: https://github.com/grpc/grpc/issues/33714
-            "grpcio != 1.56.0"
-            if sys.platform == "darwin"
-            else "grpcio",
-        ],
-        "serve": [
-            "uvicorn[standard]",
-            "requests",
-            "starlette",
-            # Tracking issue: https://github.com/tiangolo/fastapi/discussions/10948
-            "fastapi <= 0.108.0",
-            "aiorwlock",
-            "watchfiles",
-        ],
-        "tune": ["pandas", "tensorboardX>=1.9", "requests", pyarrow_dep, "fsspec"],
-        "observability": [
-            "opentelemetry-api",
-            "opentelemetry-sdk",
-            "opentelemetry-exporter-otlp",
-        ],
+        "data": [],
+        "default": [],
+        "client": [],
+        "serve": [],
+        "tune": [],
+        "observability": [],
+        "serve-grpc": [],
+        "rllib": [],
+        "train": [],
+        "air": [],
     }
-
-    # Ray Serve depends on the Ray dashboard components.
-    setup_spec.extras["serve"] = list(
-        set(setup_spec.extras["serve"] + setup_spec.extras["default"])
-    )
-
-    # Ensure gRPC library exists for Ray Serve gRPC support.
-    setup_spec.extras["serve-grpc"] = list(
-        set(
-            setup_spec.extras["serve"]
-            + [
-                "grpcio >= 1.32.0; python_version < '3.10'",  # noqa:E501
-                "grpcio >= 1.42.0; python_version >= '3.10'",  # noqa:E501
-            ]
-        )
-    )
-
-    if RAY_EXTRA_CPP:
-        setup_spec.extras["cpp"] = ["ray-cpp==" + setup_spec.version]
-
-    setup_spec.extras["rllib"] = setup_spec.extras["tune"] + [
-        "dm_tree",
-        "gymnasium==0.28.1",
-        "lz4",
-        "scikit-image",
-        "pyyaml",
-        "scipy",
-        "typer",
-        "rich",
-    ]
-
-    setup_spec.extras["train"] = setup_spec.extras["tune"]
-
-    # Ray AI Runtime should encompass Data, Tune, and Serve.
-    setup_spec.extras["air"] = list(
-        set(
-            setup_spec.extras["tune"]
-            + setup_spec.extras["data"]
-            + setup_spec.extras["train"]
-            + setup_spec.extras["serve"]
-        )
-    )
-
     setup_spec.extras["all"] = list(
         set(chain.from_iterable(setup_spec.extras.values()))
     )
@@ -334,19 +249,7 @@ if setup_spec.type == SetupType.RAY:
 # install-core-prerelease-dependencies.sh so we can test
 # new releases candidates.
 if setup_spec.type == SetupType.RAY:
-    setup_spec.install_requires = [
-        "click >= 7.0",
-        "filelock",
-        "jsonschema",
-        "msgpack >= 1.0.0, < 2.0.0",
-        "packaging",
-        "protobuf >= 3.15.3, != 3.19.5",
-        "pyyaml",
-        "aiosignal",
-        "frozenlist",
-        "requests",
-    ]
-
+    setup_spec.install_requires = []
 
 def is_native_windows_or_msys():
     """Check to see if we are running on native Windows,
@@ -781,7 +684,7 @@ setuptools.setup(
     # The BinaryDistribution argument triggers build_ext.
     distclass=BinaryDistribution,
     install_requires=setup_spec.install_requires,
-    setup_requires=["cython >= 0.29.32", "wheel"],
+    setup_requires=[],
     extras_require=setup_spec.extras,
     entry_points={
         "console_scripts": [
-- 
2.34.1

