From dec76249e3ccd2d636b80d84a52896c20ea4fc89 Mon Sep 17 00:00:00 2001
From: Matti Picus <matti.picus@gmail.com>
Date: Thu, 28 Aug 2025 23:52:06 +0300
Subject: [PATCH 04/11] update-commit-sha.patch

---
 python/ray/_version.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/ray/_version.py b/python/ray/_version.py
index b8896ef218..64d5291380 100644
--- a/python/ray/_version.py
+++ b/python/ray/_version.py
@@ -1,5 +1,5 @@
 # Replaced with the current commit when building the wheels.
-commit = "{{RAY_COMMIT_SHA}}"
+commit = "c057f1ea836f3e93f110e895029caa32136fc156"
 version = "2.49.0"
 
 if __name__ == "__main__":
-- 
2.39.5 (Apple Git-154)

