diff --git a/.bazelrc b/.bazelrc
index 1c4b9f296b..bf429439a0 100644
--- a/.bazelrc
+++ b/.bazelrc
@@ -41,6 +41,11 @@ build:clang-cl --per_file_copt="-\\.(asm|S)$@-Werror"
 build:msvc-cl     --per_file_copt="-\\.(asm|S)$@-WX"
 # Ignore warnings for protobuf generated files and external projects.
 build --per_file_copt="\\.pb\\.cc$@-w"
+# Ignore one specific warning
+build --per_file_copt="event\\.cc$@-w"
+build --per_file_copt="logging\\.cc$@-w"
+# Ignore warning when building message_lite.cc (for aarch64)
+build --per_file_copt="message_lite\.cc$@-Wno-error=stringop-overflow"
 build --per_file_copt="-\\.(asm|S)$,external/.*@-w"
 #build --per_file_copt="external/.*@-Wno-unused-result"
 # Ignore minor warnings for host tools, which we generally can't control
-- 

