From 4d5363d215b7373d207021032655282e41e9e143 Mon Sep 17 00:00:00 2001
From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
Date: Tue, 10 Nov 2020 23:26:35 +0300
Subject: [PATCH 04/13] Disable making non-core entry scripts

---
 python/setup.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index 577373e70..cb79b08ca 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -457,10 +457,10 @@ setuptools.setup(
     entry_points={
         "console_scripts": [
             "ray=ray.scripts.scripts:main",
-            "rllib=ray.rllib.scripts:cli [rllib]",
-            "tune=ray.tune.scripts:cli",
-            "ray-operator=ray.ray_operator.operator:main",
-            "serve=ray.serve.scripts:cli",
+            #"rllib=ray.rllib.scripts:cli [rllib]",
+            #"tune=ray.tune.scripts:cli",
+            #"ray-operator=ray.ray_operator.operator:main",
+            #"serve=ray.serve.scripts:cli",
         ]
     },
     include_package_data=True,
-- 
2.26.2

