From f6adc4cad5dc84ec216a806e45b618aff7577511 Mon Sep 17 00:00:00 2001
From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
Date: Tue, 10 Nov 2020 23:26:35 +0300
Subject: [PATCH 04/12] Disable making non-core entry scripts

Signed-off-by: Kai Fricke <kai@anyscale.com>
---
 python/setup.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index 957559c8b..981bad8a1 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -472,10 +472,10 @@ setuptools.setup(
     entry_points={
         "console_scripts": [
             "ray=ray.scripts.scripts:main",
-            "rllib=ray.rllib.scripts:cli [rllib]",
-            "tune=ray.tune.scripts:cli",
-            "ray-operator=ray.ray_operator.operator:main",
-            "serve=ray.serve.scripts:cli",
+            #"rllib=ray.rllib.scripts:cli [rllib]",
+            #"tune=ray.tune.scripts:cli",
+            #"ray-operator=ray.ray_operator.operator:main",
+            #"serve=ray.serve.scripts:cli",
         ]
     },
     include_package_data=True,
-- 
2.26.2

