From d8fb07467ed7b3cdf99d46ffcd86a98c6e86828f Mon Sep 17 00:00:00 2001
From: Kai Fricke <kai@anyscale.com>
Date: Fri, 3 Dec 2021 10:55:23 -0800
Subject: [PATCH 5/5] Remove all dependencies from setup.py

Signed-off-by: Jiajun Yao <jeromeyjj@gmail.com>
Signed-off-by: Matti Picus <matti.picus@gmail.com>
---
 python/setup.py | 98 ++++---------------------------------------------
 1 file changed, 8 insertions(+), 90 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index 9eef7cca46..74eb56681a 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -225,76 +225,18 @@ ray_files += [
 # also update the matching section of requirements/requirements.txt
 # in this directory
 if setup_spec.type == SetupType.RAY:
-    pandas_dep = "pandas >= 1.3"
-    numpy_dep = "numpy >= 1.20"
-    if sys.platform != "win32":
-        pyarrow_dep = "pyarrow >= 6.0.1"
-    else:
-        # Serialization workaround for pyarrow 7.0.0+ doesn't work for Windows.
-        pyarrow_dep = "pyarrow >= 6.0.1, < 7.0.0"
     setup_spec.extras = {
-        "data": [
-            numpy_dep,
-            pandas_dep,
-            pyarrow_dep,
-            "fsspec",
-        ],
-        "default": [
-            # If adding dependencies necessary to launch the dashboard api server,
-            # please add it to dashboard/optional_deps.py as well.
-            "aiohttp >= 3.7",
-            "aiohttp_cors",
-            "colorful",
-            "py-spy >= 0.2.0",
-            "requests",
-            "gpustat >= 1.0.0",  # for windows
-            "opencensus",
-            "pydantic",
-            "prometheus_client >= 0.7.1",
-            "smart_open",
-            "virtualenv >=20.0.24, < 20.21.1",  # For pip runtime env.
-        ],
-        "serve": ["uvicorn", "requests", "starlette", "fastapi", "aiorwlock"],
-        "tune": ["pandas", "tensorboardX>=1.9", "requests", pyarrow_dep],
-        "k8s": ["kubernetes", "urllib3"],
-        "observability": [
-            "opentelemetry-api",
-            "opentelemetry-sdk",
-            "opentelemetry-exporter-otlp",
-        ],
+        "data": [],
+        "default": [],
+        "serve": [],
+        "tune": [],
+        "k8s": [],
+        "observability": [],
     }
 
-    # Ray Serve depends on the Ray dashboard components.
-    setup_spec.extras["serve"] = list(
-        set(setup_spec.extras["serve"] + setup_spec.extras["default"])
-    )
-
     if RAY_EXTRA_CPP:
         setup_spec.extras["cpp"] = ["ray-cpp==" + setup_spec.version]
 
-    setup_spec.extras["rllib"] = setup_spec.extras["tune"] + [
-        "dm_tree",
-        "gymnasium==0.26.3",
-        "lz4",
-        "scikit-image",
-        "pyyaml",
-        "scipy",
-        "typer",
-        "rich",
-    ]
-
-    setup_spec.extras["train"] = setup_spec.extras["tune"]
-
-    # Ray AI Runtime should encompass Data, Tune, and Serve.
-    setup_spec.extras["air"] = list(
-        set(
-            setup_spec.extras["tune"]
-            + setup_spec.extras["data"]
-            + setup_spec.extras["train"]
-            + setup_spec.extras["serve"]
-        )
-    )
-
     setup_spec.extras["all"] = list(
         set(chain.from_iterable(setup_spec.extras.values()))
     )
@@ -307,31 +249,7 @@ if setup_spec.type == SetupType.RAY:
 # install-core-prerelease-dependencies.sh so we can test
 # new releases candidates.
 if setup_spec.type == SetupType.RAY:
-    setup_spec.install_requires = [
-        "attrs",
-        "click >= 7.0",
-        "dataclasses; python_version < '3.7'",
-        "filelock",
-        # Tracking issue: https://github.com/ray-project/ray/issues/30984
-        "grpcio >= 1.32.0, <= 1.49.1; python_version < '3.10' and sys_platform == 'darwin'",  # noqa:E501
-        "grpcio >= 1.42.0, <= 1.49.1; python_version >= '3.10' and sys_platform == 'darwin'",  # noqa:E501
-        # Original issue: https://github.com/ray-project/ray/issues/33833
-        "grpcio >= 1.32.0, <= 1.51.3; python_version < '3.10' and sys_platform != 'darwin'",  # noqa:E501
-        "grpcio >= 1.42.0, <= 1.51.3; python_version >= '3.10' and sys_platform != 'darwin'",  # noqa:E501
-        "jsonschema",
-        "msgpack >= 1.0.0, < 2.0.0",
-        "numpy >= 1.16; python_version < '3.9'",
-        "numpy >= 1.19.3; python_version >= '3.9'",
-        "packaging",
-        "protobuf >= 3.15.3, != 3.19.5",
-        "pyyaml",
-        "aiosignal",
-        "frozenlist",
-        "requests",
-        # Light weight requirement, can be replaced with "typing" once
-        # we deprecate Python 3.7 (this will take a while).
-        "typing_extensions; python_version < '3.8'",
-    ]
+    setup_spec.install_requires = []
 
 
 def is_native_windows_or_msys():
@@ -786,7 +704,7 @@ setuptools.setup(
     # The BinaryDistribution argument triggers build_ext.
     distclass=BinaryDistribution,
     install_requires=setup_spec.install_requires,
-    setup_requires=["cython >= 0.29.32", "wheel"],
+    setup_requires=[],
     extras_require=setup_spec.extras,
     entry_points={
         "console_scripts": [
-- 
2.34.1

