From fcda3379e8615b0caf8a4eaa1e4d07d2fdb87a0c Mon Sep 17 00:00:00 2001
From: Kai Fricke <kai@anyscale.com>
Date: Fri, 3 Dec 2021 11:04:33 -0800
Subject: [PATCH 5/5] Add bazel linkopts/libs

Signed-off-by: Jiajun Yao <jeromeyjj@gmail.com>
---
 python/setup.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/python/setup.py b/python/setup.py
index c6ee62a96..c5e4f839e 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -434,6 +434,8 @@ def build(build_python, build_java, build_cpp):
         raise OSError(msg)
 
     bazel_env = dict(os.environ, PYTHON3_BIN_PATH=sys.executable)
+    bazel_env.setdefault("BAZEL_LINKOPTS", "-static-libstdc++:-lm")
+    bazel_env.setdefault("BAZEL_LINKLIBS", "-l%:libstdc++.a")
 
     if is_native_windows_or_msys():
         SHELL = bazel_env.get("SHELL")
-- 
2.38.1.windows.1

