From 9204abe3efea101fc0a0e477dd124d58e012aa7e Mon Sep 17 00:00:00 2001
From: mattip <matti.picus@gmail.com>
Date: Sat, 22 Jun 2024 22:48:25 +0300
Subject: [PATCH] use bazel 6.3.2

---
 WORKSPACE | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/WORKSPACE b/WORKSPACE
index cf7d84bc88..4e33e2cd70 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -29,7 +29,7 @@ load("@bazel_skylib//lib:versions.bzl", "versions")
 
 # Please keep this in sync with the .bazelversion file.
 versions.check(
     maximum_bazel_version = "6.5.0",
-    minimum_bazel_version = "6.5.0",
+    minimum_bazel_version = "6.3.0",
 )
 
-- 
2.34.1

