From 55b18552dbb98aa0662504866dc07fa7872cfb70 Mon Sep 17 00:00:00 2001
From: Matti Picus <matti.picus@gmail.com>
Date: Sun, 23 Jul 2023 23:28:39 +0300
Subject: [PATCH] allow bazel 5.4.0

---
 WORKSPACE | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/WORKSPACE b/WORKSPACE
index f995b3b422..a27222deb1 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -22,7 +22,7 @@ load("@bazel_skylib//lib:versions.bzl", "versions")
 # it in CI.
 
 # Please keep this in sync with the .bazeliskrc file.
-versions.check(minimum_bazel_version = "5.4.1")
+versions.check(minimum_bazel_version = "5.4.0")
 
 # Tools to generate `compile_commands.json` to enable awesome tooling of the C language family.
 # Just run `bazel run @hedron_compile_commands//:refresh_all`
