From 0da8732a8d99c436c574bdbb24fe2e35df5e0a7c Mon Sep 17 00:00:00 2001
From: Jiajun Yao <jjyao@anyscale.com>
Date: Thu, 9 Dec 2021 08:45:59 -0800
Subject: [PATCH 7/7] Remove bazel version check

Signed-off-by: Jiajun Yao <jeromeyjj@gmail.com>
---
 python/setup.py | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index 565237825..80613e107 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -424,17 +424,6 @@ def build(build_python, build_java, build_cpp):
             ] + pip_packages,
             env=dict(os.environ, CC="gcc"))
 
-    version_info = bazel_invoke(subprocess.check_output, ["--version"])
-    bazel_version_str = version_info.rstrip().decode("utf-8").split(" ", 1)[1]
-    bazel_version_split = bazel_version_str.split(".")
-    bazel_version_digits = [
-        "".join(takewhile(str.isdigit, s)) for s in bazel_version_split
-    ]
-    bazel_version = tuple(map(int, bazel_version_digits))
-    if bazel_version < SUPPORTED_BAZEL:
-        logger.warning("Expected Bazel version {} but found {}".format(
-            ".".join(map(str, SUPPORTED_BAZEL)), bazel_version_str))
-
     bazel_flags = ["--verbose_failures"]
 
     if not is_automated_build:
-- 
2.30.1 (Apple Git-130)

