From baa889ae5f5867a702abbf398efe4138df28ec6b Mon Sep 17 00:00:00 2001
From: Jiajun Yao <jjyao@anyscale.com>
Date: Wed, 12 Jan 2022 10:15:14 -0800
Subject: [PATCH 8/8] Debug dashboard

Signed-off-by: Jiajun Yao <jeromeyjj@gmail.com>
---
 python/ray/_private/services.py | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/python/ray/_private/services.py b/python/ray/_private/services.py
index 8d135129b..a85dfa52f 100644
--- a/python/ray/_private/services.py
+++ b/python/ray/_private/services.py
@@ -1179,9 +1179,10 @@ def start_dashboard(require_dashboard,
         # Make sure the process can start.
         try:
             import ray.dashboard.optional_deps  # noqa: F401
-        except ImportError:
+        except ImportError as e:
             if require_dashboard:
-                raise ImportError(DASHBOARD_DEPENDENCY_ERROR_MESSAGE)
+                raise e
+                #raise ImportError(DASHBOARD_DEPENDENCY_ERROR_MESSAGE)
             else:
                 return None, None
 
-- 
2.30.1 (Apple Git-130)

