From 28d96ad79637c7f45943ba28d3139b1fdcc47f6d Mon Sep 17 00:00:00 2001
From: Matti Picus <matti.picus@gmail.com>
Date: Mon, 4 Mar 2024 15:30:15 +0200
Subject: [PATCH] shallow-clone-fails-on-windows

---
 bazel/ray_deps_setup.bzl | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/bazel/ray_deps_setup.bzl b/bazel/ray_deps_setup.bzl
index f91eb303c4..f019f8c2c2 100644
--- a/bazel/ray_deps_setup.bzl
+++ b/bazel/ray_deps_setup.bzl
@@ -258,13 +258,10 @@ def ray_deps_setup():
         url = "https://github.com/bazelbuild/rules_foreign_cc/archive/refs/tags/0.9.0.tar.gz",
     )
 
-    # Using shallow_since allows the rule to clone down fewer commits.
-    # Reference:  https://bazel.build/rules/lib/repo/git
     git_repository(
         name = "rules_perl",
         remote = "https://github.com/bazelbuild/rules_perl.git",
         commit = "022b8daf2bb4836ac7a50e4a1d8ea056a3e1e403",
-        shallow_since = "1663780239 -0700",
     )
 
     http_archive(
-- 
2.33.0.windows.2

