From 3035126de53d83ed06428f9d129b17494da3ef32 Mon Sep 17 00:00:00 2001
From: Vasilij Litvinov <vasilij.n.litvinov@intel.com>
Date: Mon, 9 Aug 2021 11:50:44 +0100
Subject: [PATCH 08/13] Add workaround for os.path.isdir on Windows

Signed-off-by: Vasilij Litvinov <vasilij.n.litvinov@intel.com>
---
 python/setup.py | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/python/setup.py b/python/setup.py
index e022c307a..db250ed3c 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -291,6 +291,25 @@ def download_pickle5(pickle5_dir):
                 wzf.close()
 
 
+def patch_isdir():
+    '''
+    Python on Windows is having hard times at telling if a symlink is
+    a directory - it can "guess" wrong at times, which bites when
+    finding packages. Replace with a fixed version which unwraps links first.
+    '''
+    orig_isdir = os.path.isdir
+    def fixed_isdir(path, debug=False):
+        orig_path = path
+        while os.path.islink(path):
+            try:
+                link = os.readlink(path)
+            except OSError as err:
+                break
+            path = os.path.abspath(os.path.join(os.path.dirname(path), link))
+        return orig_isdir(path)
+    os.path.isdir = fixed_isdir
+
+
 def replace_symlinks_with_junctions():
     _LINKS = {
         r'ray\new_dashboard': '../../dashboard',
@@ -333,6 +352,7 @@ def replace_symlinks_with_junctions():
 
 
 if is_native_windows_or_msys():
+    patch_isdir()
     replace_symlinks_with_junctions()
 
 
-- 
2.26.2

