From 495a4d5d5056d7bbf6aac1f0e6816ebc5899ee4b Mon Sep 17 00:00:00 2001
From: Jiajun Yao <jjyao@anyscale.com>
Date: Tue, 18 Jan 2022 09:40:46 -0800
Subject: [PATCH 8/8] Fix replace_symlinks_with_junctions

Signed-off-by: Jiajun Yao <jeromeyjj@gmail.com>
---
 python/setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index 80613e107..9e089329d 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -314,7 +314,7 @@ def replace_symlinks_with_junctions():
 
     # Update this list if new symlinks are introduced to the source tree
     _LINKS = {
-        r"ray\new_dashboard": "../../dashboard",
+        r"ray\dashboard": "../../dashboard",
         r"ray\rllib": "../../rllib",
         r"ray\streaming": "../../streaming/python/",
     }
@@ -353,7 +353,7 @@ def replace_symlinks_with_junctions():
                 os.path.join(os.path.dirname(path), target))
             logger.info("Setting {} -> {}".format(link, target))
             subprocess.check_call(
-                f"MKLINK /J '{os.path.basename(link)}' '{target}'",
+                f'MKLINK /J "{os.path.basename(link)}" "{target}"',
                 shell=True,
                 cwd=os.path.dirname(path))
 
-- 
2.30.1 (Apple Git-130)

