From 353efadf23e790e831f87021ae5d4517331e9a2d Mon Sep 17 00:00:00 2001
From: Kai Fricke <kai@anyscale.com>
Date: Fri, 6 Aug 2021 11:36:40 +0100
Subject: [PATCH 10/12] Empty install requirements - installed by conda build

---
 python/setup.py | 29 ++---------------------------
 1 file changed, 2 insertions(+), 27 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index a9a4fdf52..26d450a13 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -118,32 +118,7 @@ extras["all"] = list(set(chain.from_iterable(extras.values())))
 # These are the main dependencies for users of ray. This list
 # should be carefully curated. If you change it, please reflect
 # the change in the matching section of requirements/requirements.txt
-install_requires = [
-    # TODO(alex) Pin the version once this PR is
-    # included in the stable release.
-    # https://github.com/aio-libs/aiohttp/pull/4556#issuecomment-679228562
-    "aiohttp",
-    "aiohttp_cors",
-    "aioredis",
-    "click >= 7.0",
-    "colorama",
-    "dataclasses; python_version < '3.7'",
-    "filelock",
-    "gpustat",
-    "grpcio >= 1.28.1",
-    "jsonschema",
-    "msgpack >= 1.0.0, < 2.0.0",
-    "numpy >= 1.16; python_version < '3.9'",
-    "numpy >= 1.19.3; python_version >= '3.9'",
-    "protobuf >= 3.15.3",
-    "py-spy >= 0.2.0",
-    "pydantic >= 1.8",
-    "pyyaml",
-    "requests",
-    "redis >= 3.5.0",
-    "opencensus",
-    "prometheus_client >= 0.7.1",
-]
+install_requires = []
 
 
 def is_native_windows_or_msys():
@@ -520,7 +495,7 @@ setuptools.setup(
     # The BinaryDistribution argument triggers build_ext.
     distclass=BinaryDistribution,
     install_requires=install_requires,
-    setup_requires=["cython >= 0.29.15", "wheel"],
+    setup_requires=[],
     extras_require=extras,
     entry_points={
         "console_scripts": [
-- 
2.26.2

