From a9553849e50d20a0770907716a851824b78177b0 Mon Sep 17 00:00:00 2001
From: mattip <matti.picus@gmail.com>
Date: Fri, 28 Feb 2025 15:29:57 +0200
Subject: [PATCH] add-missing-include

---
 src/ray/util/pipe_logger.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/ray/util/pipe_logger.cc b/src/ray/util/pipe_logger.cc
index a75ba90dca..ac91a90cd7 100644
--- a/src/ray/util/pipe_logger.cc
+++ b/src/ray/util/pipe_logger.cc
@@ -33,6 +33,7 @@
 #include "ray/util/spdlog_fd_sink.h"
 #include "ray/util/spdlog_newliner_sink.h"
 #include "ray/util/thread_utils.h"
+#include "ray/util/util.h"
 #include "spdlog/sinks/basic_file_sink.h"
 #include "spdlog/sinks/rotating_file_sink.h"
 #include "spdlog/sinks/stdout_color_sinks.h"
-- 
2.43.0

