From 64a04a724d10186473776d2be86a72b26e752a0e Mon Sep 17 00:00:00 2001
From: Kai Fricke <kai@anyscale.com>
Date: Wed, 4 Aug 2021 11:40:39 +0100
Subject: [PATCH 10/12] Empty install requirements - installed by conda build

---
 python/setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index df87c1d42..dac4d9931 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -542,8 +542,8 @@ setuptools.setup(
     cmdclass={"build_ext": build_ext},
     # The BinaryDistribution argument triggers build_ext.
     distclass=BinaryDistribution,
-    install_requires=install_requires,
-    setup_requires=["cython >= 0.29.15", "wheel"],
+    install_requires=[],
+    setup_requires=[],
     extras_require=extras,
     entry_points={
         "console_scripts": [
-- 
2.26.2

