From 63456b06714be4011a9dcb36797b15eaed5d4bbb Mon Sep 17 00:00:00 2001
From: Jiajun Yao <jjyao@anyscale.com>
Date: Tue, 18 Jan 2022 11:54:30 -0800
Subject: [PATCH 10/10] Remove streaming

Signed-off-by: Jiajun Yao <jeromeyjj@gmail.com>
---
 BUILD.bazel     | 1 -
 python/setup.py | 1 -
 2 files changed, 2 deletions(-)

diff --git a/BUILD.bazel b/BUILD.bazel
index bfff6661c..5fdc144a1 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -2315,7 +2315,6 @@ genrule(
     name = "ray_pkg",
     srcs = [
         ":cp_raylet_so",
-        ":cp_streaming",
         ":python_sources",
         ":install_py_proto",
         ":cp_redis",
diff --git a/python/setup.py b/python/setup.py
index 158051ca0..f9206469f 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -132,7 +132,6 @@ ray_files = [
     "ray/_raylet" + pyd_suffix,
     "ray/core/src/ray/gcs/gcs_server" + exe_suffix,
     "ray/core/src/ray/raylet/raylet" + exe_suffix,
-    "ray/streaming/_streaming.so",
 ]
 
 if BUILD_JAVA or os.path.exists(
-- 
2.30.1 (Apple Git-130)

