From a63bcece5e34d4d57c57a1df8585a3956acb773d Mon Sep 17 00:00:00 2001
From: mattip <matti.picus@gmail.com>
Date: Tue, 19 Nov 2024 12:16:12 +0200
Subject: [PATCH] update spdlog version

---
 bazel/ray_deps_setup.bzl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/bazel/ray_deps_setup.bzl b/bazel/ray_deps_setup.bzl
index 86d3db4b2b..aeb652a426 100644
--- a/bazel/ray_deps_setup.bzl
+++ b/bazel/ray_deps_setup.bzl
@@ -128,8 +128,8 @@ def ray_deps_setup():
     auto_http_archive(
         name = "com_github_spdlog",
         build_file = "@com_github_ray_project_ray//bazel:BUILD.spdlog",
-        urls = ["https://github.com/gabime/spdlog/archive/v1.12.0.zip"],
-        sha256 = "6174bf8885287422a6c6a0312eb8a30e8d22bcfcee7c48a6d02d1835d7769232",
+        urls = ["https://github.com/gabime/spdlog/archive/v1.14.0.zip"],
+        sha256 = "2cd8a65885e7937fdd046b181eed7e95d61bab7006bd0fb7b9c766185ed3e0ae",
     )
 
     auto_http_archive(
-- 
2.43.0

