From a3e8904c2168c075d1bc2a7abf8f94e08d4b83a6 Mon Sep 17 00:00:00 2001
From: Austin Morton <austin.morton@aquatic.com>
Date: Mon, 1 Sep 2025 11:21:43 -0500
Subject: [PATCH] No path isolation

---
 python/setup.py | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index 82a08fda81..15b07b3660 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -634,13 +634,6 @@ def build(build_python, build_java, build_cpp):
         bazel_precmd_flags = []
         if sys.platform == "win32":
             bazel_precmd_flags = ["--output_user_root=C:/tmp"]
-        # Using --incompatible_strict_action_env so that the build is more
-        # cache-able We cannot turn this on for Python tests yet, as Ray's
-        # Python bazel tests are not hermetic.
-        #
-        # And we put it here so that does not change behavior of
-        # conda-forge build.
-        bazel_flags.append("--incompatible_strict_action_env")
 
     bazel_targets = []
     bazel_targets += ["//:gen_ray_pkg"] if build_python else []
-- 
2.43.0

