From 27bb3b1baa25830f790318234042d06039b588af Mon Sep 17 00:00:00 2001
From: mattip <matti.picus@gmail.com>
Date: Mon, 1 Sep 2025 09:20:56 +0300
Subject: [PATCH] use unzip from env

---
 BUILD.bazel | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/BUILD.bazel b/BUILD.bazel
index 804a9591b9..ebae185af6 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -412,8 +412,8 @@ genrule(
 
         tmpdir=$$(mktemp -d)
 
-        unzip -o -q $(location :core_py_proto_zip) -d "$$tmpdir"
-        unzip -o -q $(location :serve_py_proto_zip) -d "$$tmpdir"
+        $$(UNZIP) -o -q $(location :core_py_proto_zip) -d "$$tmpdir"
+        $$(UNZIP) -o -q $(location :serve_py_proto_zip) -d "$$tmpdir"
 
         files=(
             $$(ls "$$tmpdir"/ray/core/generated/*_pb2*.py)
@@ -450,7 +450,7 @@ genrule(
         set -euo pipefail
 
         rm -rf python/ray/core/generated python/ray/serve/generated
-        unzip -o -q $(location :ray_py_proto_zip) -d python
+        $$(UNZIP) -o -q $(location :ray_py_proto_zip) -d python
 
         echo "$${PWD}" > $@
     """,
@@ -469,8 +469,8 @@ genrule(
         set -euo pipefail
 
         rm -rf python/ray/core/generated python/ray/serve/generated
-        unzip -o -q $(location :ray_py_proto_zip) -d "python"
-        unzip -o -q $(location :ray_pkg_zip) -d "python"
+        $$(UNZIP) -o -q $(location :ray_py_proto_zip) -d "python"
+        $$(UNZIP) -o -q $(location :ray_pkg_zip) -d "python"
 
         echo "$${PWD}" > $@
     """,
-- 
2.43.0

