From 4080c2d6294dc466f540e1d58ef1a90317a861e0 Mon Sep 17 00:00:00 2001
From: mattip <matti.picus@gmail.com>
Date: Wed, 20 Aug 2025 11:26:01 +0300
Subject: [PATCH] update commit sha

---
 python/ray/_version.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/ray/_version.py b/python/ray/_version.py
index e07c7ea959..917bdae437 100644
--- a/python/ray/_version.py
+++ b/python/ray/_version.py
@@ -1,5 +1,5 @@
 # Replaced with the current commit when building the wheels.
-commit = "{{RAY_COMMIT_SHA}}"
+commit = "03491225d59a1ffde99c3628969ccf456be13efd"
 version = "2.48.0"
 
 if __name__ == "__main__":
-- 
2.49.0

