From 2807bbc926212d2486a8f127cb2cb07194893377 Mon Sep 17 00:00:00 2001
From: Kai Fricke <kai@anyscale.com>
Date: Fri, 6 Aug 2021 12:36:05 +0100
Subject: [PATCH 13/13] Add bazel linkopts/libs

---
 python/setup.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/python/setup.py b/python/setup.py
index a0cc87c45..77fb018d7 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -335,6 +335,8 @@ def build(build_python, build_java, build_cpp):
         raise OSError(msg)
 
     bazel_env = dict(os.environ, PYTHON3_BIN_PATH=sys.executable)
+    bazel_env.setdefault("BAZEL_LINKOPTS", "-static-libstdc++:-lm")
+    bazel_env.setdefault("BAZEL_LINKLIBS", "-l%:libstdc++.a")
 
     if is_native_windows_or_msys():
         replace_symlinks_with_junctions()
-- 
2.26.2

